nss_role.c

```diff
@@ -39,11 +39,8 @@ enum nss_status _nss_role_initgroups_dyn(char *user, gid_t main_group,
         goto libnss_role_out;
     }
 
+    /* Don't do anything on errors and try to continue operating */
     result = librole_get_directory_files(LIBROLE_CONFIG_DIR, &G);
-    if (LIBROLE_OK != result) {
```

Comment:
Consider logging the failure (like "file not found", "invalid data format", etc). It might be difficult or even inappropriate in a NSS module, though.

Reply:
Not yet. already works code.